bcda/service/service.go

```diff
@@ -369,8 +386,8 @@ func (s *service) getNewAndExistingBeneficiaries(ctx context.Context, conditions
 	//
 	oldFileUpperBound := conditions.Since
 
-	// If the _since parameter is more recent than the latest CCLF file, set the upper bound
-	// for the older file to be prior to cclfFileNew.Timestamp.
+	// If the _since parameter is more recent than the latest CCLF file timestamp, set the upper bound
+	// for the older file to be prior to the newest file's timestamp.
```

Comment:
I don't think any changes to the CCLF file queries are actually necessary here, we still want to query the lower/upper bounds based on the timestamp and not created_at. But please confirm that business logic 👋

Reply:
That is my understanding: We're tied to not using CCLF files (for current PY) generated over 45 days ago, regardless of when they actually arrived, due to the possibility of in-year re-alignments.